langchain4j-parent/pom.xml

```diff
@@ -170,6 +170,11 @@
                 <artifactId>okhttp-sse</artifactId>
                 <version>${okhttp.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.squareup.okhttp3</groupId>
+                <artifactId>logging-interceptor</artifactId>
```

Comment:
we plan to get rid of okhttp in the near future, so please dont use any okhttp-specific features

Reply:
Thanks for your review, @langchain4j!

I added this dependency to enable logging of requests and responses from SearchApi with OkHttpClient, similar to how logging is enabled in `langchain4j-web-search-engine-google-custom` via these classes:
* [`GoogleSearchApiHttpRequestLoggingInterceptor.java`](https://github.com/langchain4j/langchain4j/blob/e9a8c25df171f8fd0a16f799baa6d9439a4c3d3b/web-search-engines/langchain4j-web-search-engine-google-custom/src/main/java/dev/langchain4j/web/search/google/customsearch/GoogleSearchApiHttpRequestLoggingInterceptor.java)
* [`GoogleSearchApiHttpResponseLoggingInterceptor.java`](https://github.com/langchain4j/langchain4j/blob/e9a8c25df171f8fd0a16f799baa6d9439a4c3d3b/web-search-engines/langchain4j-web-search-engine-google-custom/src/main/java/dev/langchain4j/web/search/google/customsearch/GoogleSearchApiHttpResponseLoggingInterceptor.java)

Should I restrict the `logging-interceptor` dependency by moving it inside `langchain4j-web-search-engine-searchapi/pom.xml` instead?

Reply:
My bad, it is ok for logging, please nevermind :)